javascript/Game.js

```diff
@@ -1,16 +1,18 @@
 class Game {
-  constructor(round, player, winner) {
+  constructor(round, players, winner) {
     this.round = 0,
     this.players = [],
     this.winner = winner,
     this.categories = []
   }
 
+  createPlayers() {
+    let player1 = new Player('')
```

Comment:
This new player won't be accessible anywhere outside of the `createPlayers` method, since it's not being returned or being assigned as an instance variable. Did you maybe mean to push it into the `this.players` array for your Game instance?